packages/app/src/features/liquidity-flow/utils/get-decimal-discount.ts

```diff
@@ -0,0 +1,3 @@
+const DISCOUNT_RATIO = 10000
```

Comment:
I think, it's the same as `PERCENT_PRECISION` from `contract-constants.ts`

Reply:
Edited, thx